apps/design/frontend/src/app.tsx

```diff
@@ -74,7 +74,7 @@ const StyleOverrides = styled.div`
 
 export function App(): JSX.Element {
   return (
-    <AppBase>
+    <AppBase defaultColorMode="contrastMedium" defaultSizeMode="s">
```

Comment:
could optionally set the `legacyBaseFontSizePx` prop to the more standard `16` or `14` to avoid having large UI elements in the interim (although, I think the button controls would still be pretty large because of the 'min touch area' setting on the `s` theme 😅 )

Reply:
Yeah, I am just gonna follow up with a quick desktop size theme PR to deal with this.